packages/mask/src/social-network-adaptor/facebook.com/collecting/identity.ts

```diff
@@ -54,6 +55,16 @@ function resolveCurrentVisitingIdentityInner(
 
         const avatar = getAvatar()
 
+        if (handle && avatar) {
```

Comment:
Move this function into `packages/mask/src/social-network/ui.ts` rather than handle it for each SNS provider.

Reply:
But different SNS gets different ways of the avatar.  If we create a new collecting function, we still need handle it for each sns provider